python/README.md

```diff
@@ -32,20 +32,41 @@ In order to push work to the worker you can either:
 * a) Submit work via the AWS console, using the SQS interface
 * b) Submit work from the terminal using `aws-cli`
 
-a) Just go to the AWS console, select the relevant queue and submit your work following the format described further in this section.
+### Advanced: pushing custom work to the local worker
+
+You can also push work to a locally running `worker` instance without using the `ui` or the `api` projects.
 
-b) Make sure you have `aws-cli` installed. Then, put the work in the correct format, described further in this section in 
-a file `payload.json`. Finally, use the `aws-cli` commands to send that payload to the desired SQS queue.
-Here is an example:
+First, make sure you have InfraMock running. Then, you can use `aws-cli` to send a payload directly to the queue
+the worker is listening to:
 
     aws --endpoint-url=http://localhost:4566 sqs send-message --queue-url http://localhost:4566/queue/development-queue.fifo --message-body "$(< payload.json)" --message-group-id "$(date +%s)"
 
-which will push the payload in `payload.json` to a local worker. You can do something similar if you are connected to the
-live clusters by deleting the `--endpoint-url` argument and using the appropriate remote queue URL.
+This will push the payload in the file `payload.json` to the SQS queue the worker is listening to.
+
+
+Development
+-----------
+
+Open the Visual Studio Code workspace:
+
+    code python.code-workspace
+
+You should be prompted to run the code in a container. If not, hit `Cmd+Shift+P` and search for
+`Remote Containers: Reopen in Container`. Selecting this item will cause the Python worker to
+reopen in a container for development.
+
+### Tests
+While having the workspace running in a container, open a terminal in VS code. Go to `src/` and run:
+
+    gunzip -k ../../data/test/python.h5ad
```

Comment:
is step `gunzip -k ../../data/test/python.h5ad` necessary? I don't remember having to do it

Reply:
You need to do it the first time, after that it won't do anything. By default the files are gzipped on GitHub, so you need to unpack them.

Reply:
I see, cool!